docs/data/date-pickers/timezone/timezone.md

```diff
@@ -0,0 +1,161 @@
+|         `"default"` | Will use the default timezone of your date library, this value can be set using<br/>- [`dayjs.tz.setDefault`](https://day.js.org/docs/en/timezone/set-default-timezone) on dayjs<br/>- [`Settings.defaultZone`](https://moment.github.io/luxon/#/zones?id=changing-the-default-zone) on luxon<br/>- [`moment.tz.setDefault`](https://momentjs.com/timezone/docs/#/using-timezones/default-timezone/) on moment |
+|          `"locale"` | Will use the system's local timezone                                                                                                                                                                                                                                                                                                                                                                           |
+| IANA standard zones | Example: `"Europe/Paris"`, `"America/New_York"`<br/>[List of all the IANA zones](https://timezonedb.com/time-zones)                                                                                                                                                                                                                                                                                            |
+|        Fixed offset | Example: `"UTC+7"`<br/>**Only available with Luxon**                                                                                                                                                                                                                                                                                                                                                           |
+
+## Usage with Day.js
+
+[Dayjs timezone documentation](https://day.js.org/docs/en/timezone/timezone)
+
+### Day.js and UTC
+
+To be able to use UTC, you have to enable the `utc` plugin:
+
+```tsx
+import dayjs from 'dayjs';
+import utc from 'dayjs/plugin/utc';
+
+dayjs.extend(utc);
+```
+
+{{"demo": "DayjsUTC.js"}}
+
+### Day.js and timezones
+
+To be able to use timezones, you have to enable both the `utc` and the `timezone` plugin:
+
+```tsx
+import dayjs from 'dayjs';
+import utc from 'dayjs/plugin/utc';
+import timezone from 'dayjs/plugin/timezone';
+
+dayjs.extend(utc);
+dayjs.extend(timezone);
+```
+
+{{"demo": "DayjsTimezone.js"}}
+
+## Usage with Luxon
+
+[Luxon timezone documentation](https://moment.github.io/luxon/#/zones)
+
+### Luxon and UTC
+
+{{"demo": "LuxonUTC.js"}}
+
+### Luxon and timezone
+
+{{"demo": "LuxonTimezone.js"}}
+
+## Usage with Moment
+
+[Moment timezone documentation](https://momentjs.com/timezone/)
+
+### Moment and UTC
+
+{{"demo": "MomentUTC.js"}}
+
+### Moment and timezone
+
+To be able to use timezones, you have to pass the default export from `moment-timezone` to the `dateLibInstance` prop of `LocalizationProvider`:
```

Comment:
Like that with the display is a bit misleading with two code block around the demo, such that we don't know which codes refers to the demo.

After first reading, I thaught the following: 

> Ok, I need to pass a custom `moment` to specify the timezone. Strange that for moment we specify timezone in the `LocalizationProvider`

Then I realized the `timezone` props was still used in the next code block

Either merging the two code blocks, or adding a sentence about the next the use of `timezone` props between the code block and the demo could clarify the situation.

![image](https://user-images.githubusercontent.com/45398769/234309173-6e022249-3647-406f-b9df-c6a216a50c0b.png)

Reply:
I can't really remove the code block above because for dayjs (UTC and timezone) and for moment (timezone) it hides that you need to import a plugin, which is the most important piece of information.

Maybe I can make the code snippet more detailed and remove the preview of the demo.
Or if we have a way to put the entire code of the demo in the preview.

Reply:
@alexfauquette I'd like to discuss that point a  little more

Reply:
I reworked all the demos of "Usage with XXX" to have the "Stored values" in the demos and improve the code snippet above it to be the smallest working example possible.